fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/derby/DerbyMaster.java

```diff
@@ -46,7 +46,7 @@
     private final String database;
 
     // Controls if we run derby in debugging mode which enables more logs.
-    private final boolean debugging = false;
+    private static final boolean DEBUGGING = false;
```

Comment:
In other places, I think we just call this variable `DEBUG`.  Should we be consistent?

Reply:
👍 I'll fix in my next PR